### PR TITLE
US77483: Added a class to serialize/deserialize the response body fro…

### PR DESCRIPTION
**Pull request by anubhavi25**
…m adapters

### CODE REVIEW

File: model/src/main/java/com/ge/predix/acs/model/AdapterResponse.java [hunk 1]
```diff
@@ -0,0 +1,34 @@
+package com.ge.predix.acs.model;
```
Comment (sanjeevchopra) [inline model/src/main/java/com/ge/predix/acs/model/AdapterResponse.java]: `com.ge.predix.acs.rest.attribute.adapter`

File: model/src/main/java/com/ge/predix/acs/model/AdapterResponse.java [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+
+import java.util.Set;
+
+public class AdapterResponse {
```
Comment (sanjeevchopra) [inline model/src/main/java/com/ge/predix/acs/model/AdapterResponse.java]: `AttributesResponse` will allow for other response types from adapters in future

**Review by sanjeevchopra — DISMISSED** [review]

File: model/src/main/java/com/ge/predix/acs/model/AdapterResponse.java (hunk 1 above)
Comment (anubhavi25) [reply model/src/main/java/com/ge/predix/acs/model/AdapterResponse.java]: Done

File: model/src/main/java/com/ge/predix/acs/model/AdapterResponse.java (hunk 2 above)
Comment (anubhavi25) [reply model/src/main/java/com/ge/predix/acs/model/AdapterResponse.java]: Done

**Comment by anubhavi25** [comment]
@sanjeevchopra please do one final review so I can squash changes

**Pull request merged**